marklogic-data-hub-central/ui/src/components/queries/queries.tsx

```diff
@@ -59,17 +59,21 @@ const Query = (props) => {
                 propertiesToDisplay: props.columns,
             }
         }
-        try {
-            props.setIsLoading(true);
-            await creatNewQuery(query);
-            setOpenSaveModal(false);
-            getSaveQueries();
-        } catch (error) {
-            handleError(error);
-        } finally {
-            props.setIsLoading(false);
-            resetSessionTime();
-        }
+        props.setIsLoading(true);
+        await creatNewQuery(query);
+        setOpenSaveModal(false);
+        getSaveQueries();
+        // try {
```

Comment:
Same thing here - why not just delete this code? What's the benefit of keeping it around? It's in version history if it's ever needed again.